src/utils.ts

```diff
@@ -195,6 +195,14 @@ export const safeGet = (obj, arr, dflt = null) => {
   return current || dflt;
 };
 
+export const safeJsonParse = (obj, dflt = undefined) => {
```

Comment:
```suggestion
export const safeJsonParse = (obj, default = undefined) => {
```

Reply:
You can use `safeExecute` inside

Reply:
Moved to `safeExecute`,
But couldn't change to `default` because this is a reserved word